release-notes/3.0/preview/api-diff/preview5/3.0-preview5_System.Runtime.Loader.md

```diff
@@ -0,0 +1,15 @@
+# System.Runtime.Loader
+
+``` diff
+ namespace System.Runtime.Loader {
+     public class AssemblyLoadContext {
```

Comment:
cc @sdmaclea to verify

Reply:
What is here looks OK, but it looks like it might be incomplete.
I tried to look at the preview4 api-diff, but it is not present in the directory.
We also added:
```C#
namespace System.Runtime.Loader
{
 // Removed `abstract`, now a concrete class  
 public partial class AssemblyLoadContext
    {
        // New public constructor
        public AssemblyLoadContext(string name, bool isCollectible = false) { }
        // enumeration of all assembly load contexts
        public static System.Collections.Generic.IEnumerable<AssemblyLoadContext> All  { get { throw null; } }
        // enumeration of all assemblies in the assembly load context
        public System.Collections.Generic.IEnumerable<System.Reflection.Assembly> Assemblies  { get { throw null; } }
        // Name of the AssemblyLoadContext
        public string Name { get { throw null; } }

```

These may have been in preview4, but I didn't see them mentioned in the release notes.

Reply:
Found the preview4 diff PR.  They are in that one.

LGTM